### PR TITLE
feat: add `--file` flag (#64)

### PR DESCRIPTION
**Pull request by blackheaven**

### CODE REVIEW

File: src/NixTree/Main.hs [hunk 1]
```diff
@@ -64,6 +65,15 @@ optsParser =
                     )
               )
           )
+        <*> optional
+          ( Opts.strOption
+              ( Opts.long "file"
+                  <> Opts.metavar "FILE"
+                  <> Opts.helpDoc
+                    ( Just $ Opts.vsep ["File used by `nix path-info" ]
```
Comment (utdemir) [inline src/NixTree/Main.hs:73]: I feel like this leaks implementation details a bit - the user might not know whether/how `nix-tree` uses `nix path-info`. Maybe we should ~steal~ use their explanation instead:

```
Interpret installables as attribute paths relative to the Nix expression stored in file.
```

**Review by utdemir — APPROVED** [review]
LGTM, thanks!  Looks super useful.

Just one comment - if you can address it I'll merge it straightaway, otherwise need to wait a bit for me to get to my workstation.

File: src/NixTree/Main.hs (hunk 1 above)
Comment (blackheaven) [reply src/NixTree/Main.hs]: done

**Comment by blackheaven** [comment]
Thanks for your feedback

**Pull request merged**